tests/Luxembourg/LuxembourgTest.php

```diff
@@ -0,0 +1,92 @@
+<?php declare(strict_types=1);
+/**
+ * This file is part of the Yasumi package.
+ *
+ * Copyright (c) 2015 - 2019 AzuyaLabs
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @author Sacha Telgenhof <me@sachatelgenhof.com>
+ */
+
+namespace Yasumi\tests\Luxembourg;
+
+use ReflectionException;
+use Yasumi\Holiday;
+
+/**
+ * Class for testing holidays in Luxembourg.
+ */
+class LuxembourgTest extends LuxembourgBaseTestCase
+{
+    /**
+     * @var int year random year number used for all tests in this Test Case
+     */
+    protected $year;
+
+    /**
+     * Tests if all official holidays in Luxembourg are defined by the provider class
+     * @throws ReflectionException
+     */
+    public function testOfficialHolidays(): void
+    {
+        $this->assertDefinedHolidays([
+            'newYearsDay',
+            'easterMonday',
+            'internationalWorkersDay',
+            'europeDay',
```

Comment:
Since Europe Day is celebrated on or after 2019, you need to account for that in this test. If you use test date before 2019 your tests will fail.

Reply:
Thanks, I've changed the test, I think it's better this time ?